docroot/profiles/lagunita/sul_profile/config/sync/split/library/views.view.sul_people.yml

```
@@ -582,6 +587,203 @@ display:
         - 'config:field.storage.node.su_person_photo'
         - 'config:field.storage.node.su_person_short_title'
         - 'config:field.storage.node.su_person_type_group'
+  block_1:
+    id: block_1
+    display_title: 'Randomized Card Grid'
+    display_plugin: block
```

This is needed to differentiate it from regular blocks. `viewfield_block` allows us to more granularity pick which ones are available in the UI.
```suggestion
    display_plugin: viewfield_block
```